rca/project_styleguide/templates/patterns/pages/landingpage/landing_page--dev-all-fields.html

```diff
@@ -155,7 +155,7 @@ <h2 class="section__heading section__heading--primary heading heading--four">
 
                 </section>
             {% endif %}
-            {% if slideshow_block or page.text_teaser %}
+            {% if slideshow_block or page.cta_block %}
```

Comment:
Suggestion: Update the PL equivalent variable: https://github.com/torchbox/rca-wagtail-2019/blob/bugfix/3867-landing-page-bugs/rca/project_styleguide/templates/patterns/pages/landingpage/landing_page--dev-all-fields.yaml#L144